includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -492,6 +503,50 @@ protected function get_row_data( $item ) {
 		return $escaped_column_data;
 	}
 
+	/**
+	 * Get the date on which the last lesson was marked complete.
+	 *
+	 * @since 4.2.0
+	 *
+	 * @param array $args Array of arguments to pass to comments query.
+	 *
+	 * @return string The last activity date, or N/A if none.
+	 */
+	private function get_last_activity_date( array $args ): string {
+		$default_args  = array(
+			'number' => 1,
+			'type'   => 'sensei_lesson_status',
+			'status' => [ 'complete', 'passed', 'graded' ],
+		);
+		$args          = wp_parse_args( $args, $default_args );
+		$last_activity = Sensei_Utils::sensei_check_for_activity( $args, true );
+
+		if ( ! $last_activity ) {
+			return 'N/A';
```

Comment:
Same question as the previous comment.

Reply:
https://github.com/Automattic/sensei/pull/4879/commits/dec1456b3508583a39fab4b08848277ec34bc58d